package.json

```diff
@@ -47,13 +47,15 @@
     "@atomist/sdm": ">=1.6.0",
     "@atomist/sdm-core": ">=1.6.0",
     "@atomist/sdm-pack-build": ">=1.0.4",
+    "@atomist/sdm-pack-fingerprints": "*",
```

Comment:
Better to be more specific here, probably using the same string as the actual dependency.